integration-commons/src/main/java/org/symphonyoss/integration/model/yaml/IntegrationProperties.java

```diff
@@ -314,6 +333,10 @@ public String getIntegrationBridgeUrl() {
     return String.format("https://%s:%s/%s", integrationBridge.getHost(), port, INTEGRATION_CONTEXT);
   }
 
+  public Map<String, ConnectionInfo> getServices() { return services;  }
```

Comment:
Should you export those services as a Map object? It should be better this method returns a list of services or you might provide a method to get service by id.